siege_utilities/geo/gazetteers/wikidata_gazetteer.py

```
@@ -0,0 +1,314 @@
+        relation = elements[0]
+        coords: list[list[tuple[float, float]]] = []
+        for m in relation.get("members") or []:
+            if m.get("role") != "outer":
+                continue
+            geom = m.get("geometry") or []
+            if geom:
+                coords.append([(p["lon"], p["lat"]) for p in geom])
+        if not coords:
+            raise GazetteerBackendError(
+                f"Overpass relation {rel} has no outer ring geometry"
+            )
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)